packages/website/cypress.json

```diff
@@ -1,5 +0,0 @@
-{
-  "baseUrl": "http://localhost:3000",
-  "fixturesFolder": false,
-  "pageLoadTimeout": 600000
-}
```

Comment:
Oop, yeah, this shouldn't be here anymore haha. We can remove this separately or just lump in this PR.

Reply:
To be honest I'd rather another PR. We try to keep them granular. I just feel bad asking you to do busy work 😛. I'd sent it myself but then you wouldn't get as much contributor credit on github.com.